pythonforandroid/recipes/pygame/__init__.py

```diff
@@ -0,0 +1,56 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from os.path import join
+from pythonforandroid.toolchain import current_directory
```

Comment:
**nitpick:** Maybe we can sort the imports

Reply:
Addressed, I wish we were linting using `isort` across the code base